.github/actions/compose/docker-compose.smoketest.release.yml

```diff
@@ -36,6 +36,7 @@ services:
     environment:
       - TZ=Europe/Berlin
       - OPTIMIZE_API_ACCESS_TOKEN=secret
+      - OPTIMIZE_SUPER_USER_IDS=[demo]
```

Comment:
I remember this is a C7 only thing and will be removed. 
Does the same test rely on the demo user?

Reply:
i think I might have copied it over by mistake, will remove it